src/pymortests/demos.py

```diff
@@ -115,16 +119,16 @@
 )
 
 DEMO_ARGS = (
-    DISCRETIZATION_ARGS
-    + THERMALBLOCK_ARGS
-    + THERMALBLOCK_ADAPTIVE_ARGS
-    + THERMALBLOCK_SIMPLE_ARGS
-    + THERMALBLOCK_GUI_ARGS
-    + BURGERS_EI_ARGS
-    + PARABOLIC_MOR_ARGS
-    + SYS_MOR_ARGS
-    + HAPOD_ARGS
-    + FENICS_NONLINEAR_ARGS
+    # DISCRETIZATION_ARGS
```

Comment:
Looks like you forgot something in here.

Reply:
And I thought: 'Don't forget to remove the comments again.'